src/components/organisms/ReviewBox.tsx

```diff
@@ -0,0 +1,166 @@
+import * as S from '../../styles/organisms/ReviewBox';
+import React, { useCallback, useState } from 'react';
+import Slider from 'rc-slider';
+import 'rc-slider/assets/index.css';
+import Image from '../atoms/Image';
+import { useNavigate } from 'react-router-dom';
+// import { useMutation } from 'react-query';
+// import { PostReview } from '../../apis/review';
+import CheckboxLabel from '../molecules/CheckboxLabel';
+
+const dogOwner = [
+  '온순해요',
+  '공고대로에요',
+  '강아지 용품 준비가\n잘 되어있어요',
+  '훈련되어 있어요',
+];
+
+const partTimeWorker = [
+  '친절해요',
+  '소통이 잘 돼요',
+  '약속을 잘 지켜주셨어요',
+  '강아지를 잘 챙겨주셨어요',
+];
```

Comment:
좋은 변수명은 아닌 것 같아요.
변수에 담긴 내용을 보지 않고는 어떤 값들이 담겨있는지를 전혀 추론할 수 없어서요.
두 변수를 합쳐서 아래와 같이 만들어보는 건 어떨까요?

```suggestion
const feedbackMessage = {
  dogOwner: [
    '온순해요',
    '공고대로에요',
    '강아지 용품 준비가\n잘 되어있어요',
    '훈련되어 있어요'
  ],
  partTimeWorker: {
    '친절해요',
    '소통이 잘 돼요',
    '약속을 잘 지켜주셨어요',
    '강아지를 잘 챙겨주셨어요',
  ]
};
```

Reply:
네 수정하겠습니다!